### ISSUE TITLE
Support Hono

### ISSUE THREAD

**Issue by robertherber** [felixmosh/bull-board#621]
I've tried with all the available adapters - and none of them work with [Hono](https://hono.dev/). Hono is a barebones framework that strives to support as many runtimes by closely adhering to just web standards, and it's fast. I'm trying to build most stuff around it now - to keep the code easy to reuse and move around. But I'm using Bull a lot - so I'd really like to be able to spin up Bullboard on it as well.. :)

Is there an adapter for Hono in the works? Or is there a simpler (or more granular) way to set up Bullboard that might work?

**Comment by felixmosh** [comment]
I'm not familiar with hono, but it should be really simple to create an adapter for it.

I'll be more tham happy to review a PR gor that.
Check the express (or any other adapters) as a reference.

**Comment by felixmosh** [comment]
Thank you for sharing, maybe I can craft an official adapter out of it...

**Comment by felixmosh** [comment]
@abarakat4202 what is the difference of serveStatic from `hono/bun` that you are using, why is it "bun"?

**Comment by masterT** [comment]
It's using [Bun.file](https://bun.sh/docs/api/file-io) under the hood.

https://github.com/honojs/hono/blob/7a86bb0ef7e7fa5763643da290a2d4498d4ed269/src/adapter/bun/serve-static.ts#L10

**Comment by felixmosh** [comment]
So it is not suitable for general usage... no?

**Comment by robertherber** [comment]
Based on the example @abarakat4202 provided I added it successfully to [Zemble](https://github.com/kingstinct/zemble/blob/main/packages/bull/bullboard-hono-adapter.ts), with two slight modifications:
- I added `setNodeModulesRootPath` for monorepo support, so it's possible to adjust the path to the static files if the node_modules are hoisted at a higher level (maybe there is a better way to do this dynamically?).
- I had to remove the params from `c.html(template, params)`, with the latest version of hono it caused the html to be rendered as text.

Note: I'm also using Hono with Bun.
